### PR TITLE
[003] - Interfaces

### PR DESCRIPTION
**Pull request by yalenna**

### CODE REVIEW

File: src/Homework4/Actions.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package Homework4;
+
+public interface Actions {
+
+    void createPlant( String name, String type, String nativeRegion, boolean isWatered );
```
Comment (knadonenkoepam) [inline src/Homework4/Actions.java:5]: And again formating :) Don't use spaces when opening and closing brackets, just createPlant(String name); that's all

File: src/Homework4/HousePlant.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
+package Homework4;
+
+public class HousePlant extends Plant {
+    private String housePlantName;
```
Comment (knadonenkoepam) [inline src/Homework4/HousePlant.java:4]: These fields looks similar to me in subclasses. I'd rather move them to superclass

File: src/Homework4/Actions.java (hunk 1 above)
Comment (yalenna) [reply src/Homework4/Actions.java]: It's interesting that ctr+alt+L adding spaces in some brackets. Deleted them.

File: src/Homework4/HousePlant.java (hunk 2 above)
Comment (yalenna) [reply src/Homework4/HousePlant.java]: Moved fields to Plant.

**Review by knadonenkoepam — APPROVED** [review]

**Pull request merged**